### PR TITLE
Users validations and tests

### PR DESCRIPTION
**Pull request by hugocruz1234**
# Description

Added 2 new validations to the form so that the first and last name fields cannot be empty. Also a model test was added, to verify the optimal functioning of this one.

For more details #7

### CODE REVIEW

**Review by juperameza — APPROVED** [review]
Looks good but remember is more trustworthy to also add the not null validation to the database :rocket:

File: test/models/user_test.rb [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+
+  test "should not save user without last_name" do
+    user = User.new(first_name: "John", email: "john@example.com", password: "password")
+    assert_not user.save, "Saved the user without a last_name"
+  end
+
+  test "should save user with first_name and last_name" do
+    user = User.new(first_name: "John", last_name: "Doe", email: "john@example.com", password: "password")
+    assert user.save, "Failed to save the user with first_name and last_name"
+  end
```
Comment (JuanequeX) [inline test/models/user_test.rb:18]:
```suggestion
  class UserTest < ActiveSupport::TestCase

  def create_valid_user(first_name, last_name)
    User.new(first_name: first_name, last_name: last_name, email: "john@example.com", password: "password")
  end

  test "should not save user without first_name" do
    user = create_valid_user(nil, "Doe")
    assert_not user.valid?
    assert_includes user.errors[:first_name], "can't be blank"
  end

  test "should not save user without last_name" do
    user = create_valid_user("John", nil)
    assert_not user.valid?
    assert_includes user.errors[:last_name], "can't be blank"
  end

  test "should save user with first_name and last_name" do
    user = create_valid_user("John", "Doe")
    assert user.valid?
    assert user.save, "Failed to save the user with first_name and last_name"
  end

end
```

Comment (JuanequeX) [reply test/models/user_test.rb:18]: This avoids setting a new user in each test.

It would be a good idea to test if the user doesn't have email as well.

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: test/models/user_test.rb (hunk 1 above)
Comment (hugocruz1234) [reply test/models/user_test.rb]: Thanks for the suggestion

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**